.ci/tests/integration/e2e.yaml

```diff
@@ -9,9 +9,9 @@ setup:
         PULSAR_IMAGE_TAG=2.9.2.23 PULSAR_IMAGE=streamnative/pulsar-all KIND_PUSH=true images/build.sh
         PULSAR_IMAGE_TAG=2.9.2.23 KIND_PUSH=true images/samples/build.sh
 
-    - name: install helm
-      command: |
-        curl https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 | bash
+#    - name: install helm
```

Comment:
Why do we ignore this install helm step ?

Reply:
ahh, just for local testing, forget to revert.